### ISSUE TITLE
Was the v.3.4.0 tag moved?

### ISSUE THREAD

**Issue by l0b0** [GreycLab/CImg#422]
An [automated downstream update](https://github.com/fabaff/nixpkgs/commit/8e358c05aab7f96e4ec8d1dab493fbab124894e6) picked up a new tag and hash on 2024-06-15T05:22:32Z, but [the hash seems to have changed](https://github.com/fabaff/nixpkgs/commit/34d618b3553f6fbe1dafd4e17b2542c8b1e448ef) between then and 2024-06-16T08:20:19Z.

**Comment by dtschump** [comment]
I confirm, I made a mistake.
After releasing 3.4.0, I've fixed a minor bug during the week-end, with the idea of building new 3.4.1 pre-release packages.
But I call my build script with the wrong parameter (due to the fact that as always, I didn't type the command but searched it in the bash history, forgot about last time was not a pre-release call), thus moving tag 3.4.0.
Once the mistake done, I preferred not to touch it again.
I promise I won't touch the 3.4.0 tag again from now on : https://github.com/GreycLab/gmic/commit/ac55f66fafbe0aab0ff89ffc51f4287e05949978
Sorry about that.
